streamwebs_frontend/streamwebs/tests/forms/test_user_reg_form.py

```diff
@@ -0,0 +1,24 @@
+from django.test import TestCase
+
+from streamwebs.forms import UserForm, UserProfileForm
+
+
+class UserFormTestCase(TestCase):
+
+    def setUp(self):
+        self.expected_fields = ('username', 'email', 'password', 'first_name', 'last_name')
+
+    def test_UserForm_fields_exist(self): 
+        user_form = UserForm()
+        for i in range(len(self.expected_fields)):
+            self.assertEqual(user_form.Meta.fields[i], self.expected_fields[i])
```

Comment:
It may be a good idea to sort self.expected_fields and user_form.Meta.fields - someone could put the right fields in, but in a different order. So you could use this pattern, but use .sorted() on each array - or make each into a set(), then you could do: 
self.assertEqual(set(user_form.Meta.fields), set(self.expected_fields))
without the loop.